newIDE/app/src/ProjectCreation/CreateProjectDialog.js

```diff
@@ -67,99 +67,189 @@ export type OnCreateFromExampleShortHeaderFunction = ({|
+    initialTab
+  );
+  const [outputPath, setOutputPath] = React.useState<string>(
+    app ? findEmptyPathInDefaultFolder(app) : ''
+  );
+  const [isOpening, setIsOpening] = React.useState<boolean>(false);
+  const [newProjectName, setNewProjectName] = React.useState<string>('');
+  const [
+    selectedExampleShortHeader,
+    setSelectedExampleShortShortHeader,
+  ] = React.useState<?ExampleShortHeader>(null);
+  const [
+    preCreationDialogOpen,
+    setPreCreationDialogOpen,
+  ] = React.useState<boolean>(false);
+
+  const openPreCreationDialog = React.useCallback((open: boolean) => {
+    if (open) {
+      setOutputPath(app ? findEmptyPathInDefaultFolder(app) : '');
+      setNewProjectName(generateName());
+    }
+    setPreCreationDialogOpen(open);
+  }, []);
+
+  const actions = React.useMemo(
+    () => [
+      <RaisedButton
+        key="create-blank"
+        label={<Trans>Create a blank project</Trans>}
+        primary={false}
+        onClick={() => openPreCreationDialog(true)}
+      />,
+      <FlatButton
+        key="close"
+        label={<Trans>Close</Trans>}
+        primary={false}
+        onClick={onClose}
+      />,
+    ],
+    [onClose, openPreCreationDialog]
+  );
 
-  _onChangeTab = (newTab: CreateProjectDialogTabs) => {
-    this.setState({
-      currentTab: newTab,
-    });
+  const secondaryActions = React.useMemo(
+    () => [
+      currentTab === 'games-showcase' ? (
+        <FlatButton
+          key="submit-game-showcase"
+          onClick={() => {
+            Window.openExternalURL(
+              'https://docs.google.com/forms/d/e/1FAIpQLSfjiOnkbODuPifSGuzxYY61vB5kyMWdTZSSqkJsv3H6ePRTQA/viewform?usp=sf_link'
+            );
+          }}
+          primary
+          icon={<PublishIcon />}
+          label={<Trans>Submit your game to the showcase</Trans>}
+        />
+      ) : null,
+      currentTab === 'examples' ? (
+        <FlatButton
+          key="submit-example"
+          onClick={() => {
+            Window.openExternalURL(
+              'https://github.com/GDevelopApp/GDevelop-examples/issues/new/choose'
+            );
+          }}
+          primary
+          icon={<PublishIcon />}
+          label={<Trans>Submit your project as an example</Trans>}
+        />
+      ) : null,
+    ],
+    [currentTab]
+  );
+
+  if (!open) return null;
+
+  const createProject = async (i18n: I18nType) => {
+    setIsOpening(true);
+
+    try {
+      let projectMetadata;
+
+      if (selectedExampleShortHeader) {
+        projectMetadata = await onCreateFromExampleShortHeader({
+          i18n,
+          outputPath,
+          projectName: newProjectName,
+          exampleShortHeader: selectedExampleShortHeader,
+        });
+      } else {
+        projectMetadata = await onCreateBlank({
+          i18n,
+          outputPath,
+          projectName: newProjectName,
+        });
+      }
+
+      if (!projectMetadata) return;
```

Comment:
I'd add a console.error as well here, this should not happen

Reply:
In both function `onCreateBlank` et `onCreateFromExampleShortHeader` they display error boxes if an error is thrown, with the raw error.